queries/omen/dangerKeywords.ts

```diff
@@ -0,0 +1,37 @@
+export const dangerKeywords = [
+  'dead',
+  'kill',
+  'assassinated',
+  'murder',
+  'slaughter',
+  'massacre',
+  'attack',
+  'terrorist',
+  'bomb',
+  'violence',
+  'suicide',
+  'explosion',
+  'genocide',
+  'bloodshed',
+  'execution',
+  'hostage',
+  'kidnap',
+  'stab',
+  'shoot',
+  'death',
+  'torture',
+  'arson',
+  'abduction',
+  'rape',
+  'assault',
+  'riot',
+  'ambush',
+  'strangle',
+  'lynch',
+  'behead',
+  'decapitate',
+  'mass shooting',
+  'homicide',
+  'manslaughter',
+  'assassination',
+];
```

Comment:
WDYT about alphabetically sorting this list + updating the list name? It would make it easier to maintain (find keywords to add/delete them)

```suggestion
export const dangerousKeywords = [
  'abduction', // -> Maybe I'd remove this and keep `kidnap`
  'ambush', // -> not sure this might be a danger word
  'arson',
  'assassinated',
  'assassination',
  'assault',
  'attack',
  'behead',
  'bloodshed',
  'bomb',
  'dead',
  'death',
  'decapitate',
  'execution',
  'explosion', // -> not sure this might be a dangerous word, could filter markets related to cases related to gas explosions
  'genocide',
  'homicide',
  'hostage',
  'kidnap',
  'kill',
  'lynch',
  'manslaughter',
  'mass shooting',
  'massacre',
  'murder',
  'rape',
  'riot', // -> I'd remove this, as it might filter anything related to Riot Games
  'slaughter',
  'suicide',
  'stab',
  'shoot', // -> I'd remove this as it might filter out movies-related markets
  'strangle',
  'terrorist',
  'torture',
  'violence', // -> I'd remove this as it might filter out yearly social statistics markets
];
```